src/radiation/kd_grid.hpp

```diff
@@ -0,0 +1,77 @@
+// © 2021. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract
+// 89233218CNA000001 for Los Alamos National Laboratory (LANL), which
+// is operated by Triad National Security, LLC for the U.S.
+// Department of Energy/National Nuclear Security Administration. All
+// rights in the program are reserved by Triad National Security, LLC,
+// and the U.S. Department of Energy/National Nuclear Security
+// Administration. The Government is granted for itself and others
+// acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works,
+// distribute copies to the public, perform publicly and display
+// publicly, and to permit others to do so.
+
+#ifndef RADIATION_KD_GRID_HPP_
+#define RADIATION_KD_GRID_HPP_
+
+#include <parthenon/driver.hpp>
+#include <parthenon/package.hpp>
+#include <utils/error_checking.hpp>
+using namespace parthenon::driver::prelude;
+using namespace parthenon::package::prelude;
+using namespace parthenon;
+
+#include "geometry/geometry.hpp"
+
+namespace radiation {
+namespace kdgrid {
+
+KOKKOS_INLINE_FUNCTION
+void integrate_ninj_domega_quad(const Real xi0, const Real xi1, const Real phi0,
+                                const Real phi1, Real *wgts) {
+  // expressions validated through numerical quadrature in python
```

Comment:
can we put that python script somewhere in the codebase?

Reply:
@jdolence